### DISCUSSION TITLE
How can I recognize a locally installed wallet app in the Android device's browser after web integration?

### DISCUSSION THREAD

**Discussion by NathansLiu** [WalletConnect/walletconnect-monorepo#3687] [Developer Support]
Clicking "Select Wallet," I can recognize BitKeep or other apps installed on my Android phone, but I can't identify my WalletApp. Do I need to configure it separately somewhere?

**Comment by glitch-txs** [comment]
Hi, have you submitted your wallet form in your cloud's project?

**Comment by mshahjahan-mufin** [comment]
@NathansLiu Complete the project setup on WalletConnect console and you will start getting your wallet in the list once your porject is approved
